webapi/tct-colors-css3-tests/tests.full.xml

```diff
@@ -1293,5 +1281,20 @@
         </specs>
       </testcase>
     </set>
+    <set name="color-bdd" type="js" ui-auto="bdd">
+      <testcase component="W3C_HTML5 APIs/DOM, Forms and Styles/CSS Color Module Level 3" execution_type="manual" id="css3_colors_tests_entry1" priority="P1" purpose="Check that color: currentColor acts like inherit." status="approved" type="compliance">
+        <description>
+          <test_script_entry>/opt/tct-colors-css3-tests/bdd_file/t31-color-currentColor-b.htm</test_script_entry>
```

Comment:
please don't move these html file under "bdd_file", and follow our rules discussed to put them